evm_loader/solana_utils.py

```diff
@@ -225,7 +225,7 @@ def __create_storage_account(self, seed):
     def __create_instruction_data_from_tx(self, ethereum_transaction):
         self.__create_solana_ether_caller(ethereum_transaction)
         caller_trx_cnt = getTransactionCount(client, ethereum_transaction._solana_ether_caller)
-        trx_raw = {'to': solana2ether(ethereum_transaction.contract_account),
+        trx_raw = {'to': None, # TODO FIX IT #solana2ether(ethereum_transaction.contract_account),
```

Comment:
@vasiliy-zaznobin you made this, so fix it by yourself.
This piece of software is beyond of my ability to comprehend.

Reply:
Ok, no need to fix it.
I will completely remove this tomorrow, alongside with external call functionality

Reply:
As I can see this functionality is used only in external_call tests. This functionality will be removed in #269, so we can not spend time on these tests now.